packages/element-library/src/video/playPauseButton.js

```diff
@@ -216,13 +221,9 @@ function PlayPauseButton({
 
     if (isPlaying) {
       videoNode.pause();
-      setIsPlaying(false);
       setShowControls(true);
     } else {
-      videoNode
-        .play()
-        .then(() => setIsPlaying(true))
-        .catch(() => {});
+      videoNode.play();
```

Comment:
Same here as in https://github.com/GoogleForCreators/web-stories-wp/pull/11067#discussion_r834356976

`.play()` returns a Promise, which means it can get rejected. Which is why we'd need the `catch(() => {})` here to avoid throwing errors.

See https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/play

Reply:
Good catch!

Worth noting there are a couple other place without a catch. 

https://github.com/GoogleForCreators/web-stories-wp/blob/e00e50da7a02d956b993f4038ca1c15d46eda3ef/packages/story-editor/src/components/videoTrim/useVideoNode.js#L57

https://github.com/GoogleForCreators/web-stories-wp/blob/e00e50da7a02d956b993f4038ca1c15d46eda3ef/packages/story-editor/src/components/videoTrim/useVideoNode.js#L78

Reply:
If you could create a ticket for that & assign to Prometheus pod, that would be great!

Reply:
Ticket created https://github.com/GoogleForCreators/web-stories-wp/issues/11085